app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/Utils/Networking.java

```diff
@@ -42,7 +54,64 @@ public static JSONObject getJSON(String append){
             e.printStackTrace();
             return null;
         }
+    }
+
+     @RequiresApi(api = Build.VERSION_CODES.KITKAT)
+     public static JSONObject postJSON(String append, HashMap<String, String> params){
```

Comment:
`...params) {`

Reply:
Tried that, caused some error the the code further down the line. I'll avoid touching that as Jonvi knows that section a lot better than I do.